tests/test_backbone_common.py

```diff
@@ -0,0 +1,175 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import inspect
+
+from transformers.testing_utils import require_torch, torch_device
+
+
+@require_torch
+class BackboneTesterMixin:
+    all_model_classes = ()
+    has_attentions = True
+    is_training = False
+    test_pruning = False
+    test_resize_embeddings = False
+    test_head_masking = False
```

Comment:
We don't need these 4 attributes, I think.

Reply:
Yep! Removed